pkg/rhcos/gcp.go

```diff
@@ -0,0 +1,37 @@
+package rhcos
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+)
+
+// URL fetches the location of the public GCP storage bucket containing the RHCOS image
+func URL(ctx context.Context) (string, error) {
```

Comment:
I think this should be called GCP 
`rhcos.GCP` becomes the API for getting GCP type image source.